woocommerce/woocommerce-gateway-stripe
# Remove animated supported credit card icons

Fixes #2033

# Changes proposed in this Pull Request:

- Revert part of #1979.
- Remove payment icon not only from UPE, but also from the credit card payment method, like in WCPay.

# Testing instructions

- Ensure UPE is disabled.
- Add a product to the cart.
- Go to the checkout page and notice there are no icons next to "Credit card (Stripe)".
